mods/dye/init.lua

```diff
@@ -1,23 +1,28 @@
+-- dye/init.lua
+
 dye = {}
 
+-- Load support for MT game translation.
+local S = minetest.get_translator() 
```

Comment:
```suggestion
local S = minetest.get_translator()
```
This is one of the warnings of travis.

Reply:
THX